src/librustc_mir/transform/qualify_min_const_fn.rs

```diff
@@ -390,8 +390,10 @@ fn check_terminator(
             cleanup: _,
         } => check_operand(tcx, cond, span, def_id, body),
 
-        TerminatorKind::FalseUnwind { .. } => {
+        TerminatorKind::FalseUnwind { .. } if !tcx.features().const_loop => {
```

Comment:
Just to be clear, like with const control flow, we allow these to be used in stable const fn because libcore can turn on this feature gate. There's no `allow_internal_unstable` necessary as far as I can tell

Reply:
Me & Oliver discussed this privately on Discord with the conclusion that we shouldn't allow this and that we should fix this for `const_if_match` and `const_mut_refs` as well, possibly with a beta-backport where necessary.

Reply:
See https://github.com/rust-lang/rust/pull/67251 for how to correctly feature gate it